cabal.project

```diff
@@ -158,12 +158,6 @@ source-repository-package
     location: https://gitlab.com/twittner/wai-routing
     tag: 7e996a93fec5901767f845a50316b3c18e51a61d
 
-source-repository-package
-    type: git
-    location: https://github.com/wireapp/tasty.git
-    tag: 394943c7672e5ad269e5587528b7678caf3b0720
-    subdir: hunit
-
```

Comment:
Why has this disappeared now?

Reply:
just ran the script without thinking about it :)

i shall investigate!

Reply:
i think this has been added to cabal at some point in the past, without having been added to stack.yaml.  is that plausible?  fix coming up!

(what's the reason we keep stack.yaml around, btw?  is there a ticket to remove it?  does anybody know?)

Reply:
I thought we had completely removed stack by now...